src/radio-tile/radio-tile.scss

```
@@ -0,0 +1,143 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+@mixin iui-radio-tile {
+  cursor: pointer;
+
+  &:first-of-type > .iui-radio-tile-content {
+    border-radius: $iui-border-radius 0 0 $iui-border-radius;
+  }
+
+  &:last-of-type > .iui-radio-tile-content {
+    border-radius: 0 $iui-border-radius $iui-border-radius 0;
+  }
+
+  &:not(:first-of-type) > .iui-radio-tile-content {
+    margin-left: -1px;
+  }
+
+  &-content {
+    width: $iui-xl * 5;
+    height: 100%;
+    box-sizing: border-box;
+    padding: $iui-s;
+    position: relative;
+    z-index: 1;
+    border: 1px solid rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-4));
+    background-color: var(--iui-color-background-1);
+    @media (prefers-reduced-motion: no-preference) {
+      transition: border-color $iui-speed-fast ease-out;
+    }
+  }
+
+  &-container {
+    display: inline-flex;
+    flex-wrap: wrap;
+    user-select: none;
+    // Create stacking context
+    position: relative;
+    z-index: 0;
+  }
+
+  &-input {
+    width: 0;
+    height: 0;
+    appearance: none;
+    opacity: 0;
+    position: absolute;
+    @include focus-visible {
+      + * {
+        outline: 2px solid var(--iui-color-foreground-primary);
+        outline-offset: -4px;
+      }
+    }
+
+    + *:hover {
+      z-index: 2;
+      border-color: rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-2));
+
+      > * {
+        fill: var(--iui-icons-color-actionable);
+      }
+    }
+
+    &:checked + * {
+      padding: $iui-s - 1;
+      z-index: 3;
+      border: 2px solid var(--iui-color-foreground-primary);
+
+      > * {
+        fill: var(--iui-icons-color-primary);
+      }
+
+      &::after {
+        content: '';
+        width: $iui-icons-default;
+        height: $iui-icons-default;
+        position: absolute;
+        top: $iui-s;
+        right: $iui-s;
+        background-color: var(--iui-color-foreground-primary);
+        mask: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path d="M6,14L0,8l2-2l4,4l8-8l2,2L6,14z" /></svg>');
+      }
+    }
+
+    &:disabled + * {
+      cursor: not-allowed;
+      z-index: 0;
+      border-color: var(--iui-color-background-disabled);
+      background-color: var(--iui-color-background-disabled);
+
+      &::after {
+        background-color: var(--iui-icons-color-actionable-disabled);
+      }
+
+      .iui-radio-tile-icon {
+        fill: var(--iui-icons-color-actionable-disabled);
+        filter: $iui-icons-color-multicolor-disabled;
+      }
+
+      .iui-radio-tile-label {
+        color: var(--iui-text-color-muted);
+      }
+    }
+
+    // Styles disabled checked radio tile
+    &:disabled:checked + * {
+      z-index: 3;
+      border: 2px solid rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-4));
```

```suggestion
      border: 2px solid rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-4));
      @media (forced-colors: active) {
        border-color: GrayText;
      }
```
Adds support for `forced-colors`.